### PR TITLE
HW2 Implement my own Array methods

### PR DESCRIPTION
**Pull request by Captain-Vorobey**
Check my readme

Screenshot of results in console:
- image

### CODE REVIEW

**Comment by vladhilko** [comment]
You removed other tasks inside your pr, could you return them back? 

Screenshot 2021-04-22 at 14 19 32

File: HW02/vorobyov_egor/my_array.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  include Func
+
+  def initialize(arr = [])
+    @arr = arr
+  end
+
+  def each(&block)
+    @arr.each do |el|
+      block.call(el)
+    end
```
Comment (vladhilko) [inline HW02/vorobyov_egor/my_array.rb:13]: Screenshot 2021-04-22 at 14 22 20

**Comment by Captain-Vorobey** [comment]
> You removed other tasks inside your pr, could you return them back?
> 
> Screenshot 2021-04-22 at 14 19 32

These are decisions of other people which I pushed and didn't notice. I woun't push it back, you also can see what were there.

**Comment by vladhilko** [comment]
If you click on `file changed` you will see that your changes remove other works and if I merge it than everything will be deleted in the main repo. Your changes should only include files with your code

Screenshot 2021-04-22 at 22 15 24

File: HW02/vorobyov_egor/my_array.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+class MyArray
+  include Func
+
+  def initialize(arr = [])
+    @arr = arr
+  end
+
+  def each(&block)
+    @arr.each do |el|
```
Comment (B0RlS) [inline HW02/vorobyov_egor/my_array.rb:11]: same comment https://github.com/labs-ruby/homeworks-2021/pull/9/files#r618312721

**Review by B0RlS — APPROVED** [review]
LGTM, but there is one remark
I advise you next time, give more informative commit names.

**Pull request merged**